stko/molecular/conversion/utilities.py

```diff
@@ -0,0 +1,156 @@
+"""
+Utilities for molecule conversion.
+
+"""
+
+import logging
+import numpy as np
+from ase.io import write
+from .ase import ASEConverter
+
+from ...utilities import periodic_table
+
+logger = logging.getLogger(__name__)
+
+
+def write_cif(mol, unit_cell, filename):
```

Comment:
I don't think this function is necessary. The user can use the ASE interface to write the CIF in a single line of code

Reply:
It is a utility because an Optimizer in a different PR requires it, otherwise yes, I would not have included it.